packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * External dependencies
+ */
+import { first, last } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useCallback } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import { __ } from '@wordpress/i18n';
+
+function KeyboardShortcuts() {
+	// Registering the shortcuts
+	const { registerShortcut } = useDispatch( 'core/keyboard-shortcuts' );
+	useEffect( () => {
+		registerShortcut( {
+			name: 'core/block-editor/duplicate',
+			category: 'block',
+			description: __( 'Duplicate the selected block(s).' ),
+			keyCombination: {
+				modifier: 'primaryShift',
+				character: 'd',
+			},
+		} );
+
+		registerShortcut( {
+			name: 'core/block-editor/remove',
+			category: 'block',
+			description: __( 'Remove the selected block(s).' ),
+			keyCombination: {
+				modifier: 'access',
+				character: 'z',
+			},
+		} );
+
+		registerShortcut( {
+			name: 'core/block-editor/insert-before',
+			category: 'block',
+			description: __( 'Insert a new block before the selected block(s).' ),
+			keyCombination: {
+				modifier: 'primaryAlt',
+				character: 't',
+			},
+		} );
+
+		registerShortcut( {
+			name: 'core/block-editor/insert-after',
+			category: 'block',
+			description: __( 'Insert a new block after the selected block(s).' ),
+			keyCombination: {
+				modifier: 'primaryAlt',
+				character: 'y',
+			},
+		} );
+
+		registerShortcut( {
+			name: 'core/block-editor/delete-multi-selection',
+			category: 'block',
+			description: __( 'Remove multiple selected blocks.' ),
+			keyCombination: {
+				character: 'del',
+				aliases: [
+					'backspace',
+				],
+			},
+		} );
```

Comment:
Delete is working now, but it looks like backspace isn't. It looks like the shape of the shortcut object is expected to be different:
```js
		registerShortcut( {
			name: 'core/block-editor/delete-multi-selection',
			category: 'block',
			description: __( 'Remove multiple selected blocks.' ),
			keyCombination: {
				character: 'del',
			},
			aliases: [
				{ character: 'backspace' },
			],
		} );
```

Reply:
I fixed the format but I noticed that it doesn't work still. I think it's an event bubbling issue. It's present in master too.

Reply:
It looks like `aliases` isn't in the right place—it needs to be moved out of the `keyCombination` object.

Reply:
Not sure I understand?

Reply:
Oh nvm :)